shapely/tests/test_constructive.py

```diff
@@ -57,6 +57,13 @@
 @pytest.mark.parametrize("geometry", all_types)
 @pytest.mark.parametrize("func", CONSTRUCTIVE_NO_ARGS)
 def test_no_args_array(geometry, func):
+    if (
+        geometry.is_empty
+        and shapely.get_num_geometries(geometry) > 0
+        and func is shapely.node
+        and shapely.geos_version < (3, 8, 3)
+    ):
+        pytest.xfail("GEOS < 3.8.3 crashes with empty geometries")  # GEOS GH-601
```

Comment:
this test logic is here because `MIN_GEOS_VERSION = "3.8"` implies 3.8.0, which is a version still widely used, e.g. with [Ubuntu focal (20.04LTS)](https://packages.ubuntu.com/focal/libgeos-3.8.0) (also the main OS for my laptop).